cpp/src/lists/lists_column_factories.cu

```diff
@@ -45,7 +45,7 @@ std::unique_ptr<column> make_lists_column(size_type num_rows,
   return std::make_unique<column>(cudf::data_type{type_id::LIST},
                                   num_rows,
                                   rmm::device_buffer{0, stream, mr},
-                                  null_mask,
+                                  std::forward<rmm::device_buffer>(null_mask),
```

Comment:
You only need `std::forward` when you're working with a forwarding reference (e.g., `template <typename T> (T&&)`). Here, it's just an r-value ref, so you can just use move. 
```suggestion
                                  std::move<rmm::device_buffer>(null_mask),
```

Reply:
Thanks. Fixed.